llvm-runtimes/libgcc/libgcc-19.1.4.ebuild

```diff
@@ -0,0 +1,47 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit crossdev llvm.org toolchain-funcs
+
+DESCRIPTION="Compiler runtime library for clang, compatible with libgcc_s"
+HOMEPAGE="https://llvm.org/"
+
+LICENSE="Apache-2.0-with-LLVM-exceptions || ( UoI-NCSA MIT )"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~loong ~mips ~ppc64 ~riscv ~x86 ~amd64-linux ~arm64-macos ~ppc-macos ~x64-macos"
+
+DEPEND="
+	llvm-runtimes/compiler-rt:${LLVM_MAJOR}
+	=llvm-runtimes/libunwind-${PV}
+	!!sys-devel/gcc
+"
+BDEPEND="
+	llvm-core/clang:${LLVM_MAJOR}
+"
+
+LLVM_COMPONENTS=( llvm-libgcc )
+llvm.org_set_globals
+
+src_compile() {
+	$(tc-getCC) -E -xc \
+		"${WORKDIR}/llvm-libgcc/gcc_s.ver.in" \
+		-o gcc_s.ver || die
+	$(tc-getCC) -nostdlib \
+		-Wl,--version-script,gcc_s.ver \
+		-Wl,--whole-archive \
+		"${EPREFIX}/usr/lib/clang/${LLVM_MAJOR}/lib/linux/libclang_rt.builtins-${CTARGET%%-*}.a" \
+		"${EPREFIX}/usr/lib/libunwind.a" \
+		-Wl,-soname,libgcc_s.so.1.0 \
+		-lc -shared \
+		-o libgcc_s.so.1.0 || die
+}
+
+src_install() {
+	dolib.so libgcc_s.so.1.0
+	dosym "clang/${LLVM_MAJOR}/lib/linux/libclang_rt.builtins-${CTARGET%%-*}.a" /usr/lib/libgcc.a
+	dosym libgcc_s.so.1.0 /usr/lib/libgcc_s.so.1
+	dosym libgcc_s.so.1 /usr/lib/libgcc_s.so
+	dosym libunwind.a /usr/lib/libgcc_eh.a
```

Comment:
All `get_libdir` (put it in a variable).

Reply:
Done